src/convertToObject.js

```diff
@@ -3,16 +3,25 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceObject = {};
+  const sourceArray = sourceString.split(';');
+
+  sourceArray.filter(style => style
+    .includes(':'))
+    .forEach(element => {
```

Comment:
1. I don't see that this `filter` is needed here. All of the strings inside the array have `:`
2. Formatting
```
  sourceArray
    .filter(style => style.includes(':'))
    .forEach(element => {